--- ascal-0.1.1/configure.ac.orig	2006-12-30 01:54:39.000000000 +0100
+++ ascal-0.1.1/configure.ac	2008-01-05 16:52:13.008151938 +0100
@@ -61,11 +61,7 @@
 
 if test "$enable_debug" = yes; then
         AC_DEFINE(DEBUG, 1, [Enable debug msg.])
-		CPPFLAGS=[`echo "$CXXFLAGS" | $SED -e 's/-O[0-9]*//g'`]
-        CPPFLAGS="$CPPFLAGS -g -O0"
-else
-    CPPFLAGS=[`echo "$CXXFLAGS" | $SED -e 's/-O[0-9]*//g'`]
-	CPPFLAGS="$CPPFLAGS -O3"
+        CXXFLAGS="$CXXFLAGS -g"
 fi
 
 AC_ARG_ENABLE(Werror,
@@ -127,7 +123,7 @@
 echo "  Compile Warnings: $enable_Werror"
 echo ""
 echo "The following options will be passed to the compiler:"
-echo "  $CPPFLAGS"
+echo "  $CPPFLAGS $CXXFLAGS"
 echo ""
 echo "======================================================================"
 echo ""
